### PR TITLE
Numpy Frontend Argmax 

### PR DESCRIPTION
**Pull request by westrany**
Define argmax, return np_frontend.argmax

### CODE REVIEW

**Comment by ogbanugot** [comment]
Hi @westrany 🙂, I have linked this to the corresponding issue, but in future you should do this yourself by adding a comment ‘Close #Issue_number’, as explained [here](https://lets-unify.ai/ivy/contributing/1_the_basics.html#todo-list-issues)”

**Comment by ogbanugot** [comment]
Hi @westrany 👋 , good work so far, but you should add a test for the function. You can refer to the docs [here](https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#frontend-instance-method-test-examples) for how to write the test.

**Review by ogbanugot — CHANGES_REQUESTED** [review]
Good work so far, but you should add a test for the function. You can refer to the docs [here](https://lets-unify.ai/ivy/deep_dive/17_ivy_frontends_tests.html#frontend-instance-method-test-examples) for how to write the test. Feel free to request for a review once done.

**Comment by westrany** [comment]
@ogbanugot added argmax test in test_ndarray.py. Could I get a review?
close #5171

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py [hunk 1]
```diff
@@ -4,10 +4,48 @@
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
-from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args, num_positional_args
 import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
 
 
+# argmax
+handle_cmd_line_args
```
Comment (ogbanugot) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:12]: Hey! Good work on the test, this should be `@handle_cmd_line_args`

**Review by ogbanugot — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py (hunk 1 above)
Comment (westrany) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:12]: Hi, thanks for spotting that! Fixed it :)

Comment (ogbanugot) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py:12]: Hi, I don't think you committed the changes though, it's still the same. Kindly confirm. Thanks!

Comment (westrany) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py]: Hi, it should be fixed now. Let me know if there's anything else I need to change!

**Comment by ogbanugot** [comment]
Hey! @westrany Everything looks good but you have one file committed that seems to have commits from an old branch or fork, not sure. If you check `Files changed` you would see this file `ivy_tests/test_array_api` can you remove those changes and push. You should have only two files changed.

**Comment by westrany** [comment]
Hi @ogbanugot I think it should be fixed now

**Comment by ogbanugot** [comment]
Great job @westrany

**Review by ogbanugot — APPROVED** [review]

**Pull request merged**